wiki/Main_Page/en.md

```diff
@@ -3,17 +3,20 @@ layout: "main_page"
 ---
 
 <div class="wiki-main-page__blurb">
-The osu! wiki is a project based on the contributions of people from osu!. The end goal is that it replaces the old FAQ section, and allows it to become a much more exhaustive resource. However, to make this possible, we need help from everyone willing to contribute!
+Welcome to osu!, a free rhythm game created by peppy with its primary focus of clicking circles along with the beat.
```

Comment:
Is the line break here intentional, if not, please merge the two lines together.

Reply:
if you mean the one that's seperated by one newline its intended since it presents a different information from the "Welcome introduction".

Reply:
No, I was talking about lines 6 and 7 prior to 3284ef7. The way it is now is okay.

Reply:
Ah thanks, I also saw it but I wasn't sure which one.